Makefile

```diff
@@ -1,7 +1,7 @@
 .PHONY: test install
 
 install:
-	go get -t -v ./...
+	go get -t -v
 
 test: install
-	go test -race -cover -v ./...
```

Comment:
Why are you removing the `./...`?

Reply:
I understand now, this change was intentional due to the subfolder, if we remove the subfolder altogether, we can revert this change.